.circleci/src/jobs/deliver_web.yml

```diff
@@ -44,16 +44,16 @@ steps:
             command: sftp -b - -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/webnext.integreat.app \<<< "put -r $HOME/attached_workspace/dist/integreat-test-cms/*"
         - run:
             name: Malte development delivery
-            command: sftp -b - -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/webnext.malteapp.de \<<< "put -r $HOME/attached_workspace/dist/malte/*"
+            command: sftp -b - -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/webnext.malteapp.de \<<< "put -r $HOME/attached_workspace/dist/malte-test-cms/*"
         - run:
             name: Aschaffenburg development delivery
             command: sftp -b - -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/webnext.aschaffenburg.app \<<< "put -r $HOME/attached_workspace/dist/aschaffenburg/*"
         - run:
             name: 'Sentry Upload: integreat-test-cms'
             command: tools/sentry-release web-integreat-test-cms "${NEW_VERSION_NAME}" ~/attached_workspace/dist/integreat-test-cms
         - run:
-            name: 'Sentry Upload: malte'
-            command: tools/sentry-release web-malte "${NEW_VERSION_NAME}" ~/attached_workspace/dist/malte
+            name: 'Sentry Upload: malte-test-cms'
+            command: tools/sentry-release web-malte-test-cms "${NEW_VERSION_NAME}" ~/attached_workspace/dist/malte-test-cms
```

Comment:
@maxammann do I have to set up anything in sentry for this?

Reply:
I dont think so